example/fixtures.py

```diff
@@ -13,15 +13,11 @@
 
```

Comment:
Should this still be called `fixtures.py` if we don't actually have any (pytest) fixtures in it? `resources.py`? (Maybe that is worse)

Reply:
Maybe `definitions.py`?